frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -677,6 +677,14 @@ void main(List<String> args) async {
       final resp = await api.duration(d: duration);
       expect(resp.inHours, duration.inHours);
     });
+
+    test('Combined Chrono types', () async {
+      final test = await api.testChrono();
+      expect(castInt(test.dt!.millisecondsSinceEpoch), castInt(1631297333000));
+      expect(castInt(test.dt2!.millisecondsSinceEpoch), castInt(1631297333000));
+      expect(test.du, Duration(hours: 4));
```

Comment:
uber-nit: would be great to add one more test, about is accuracy. For example, if your design makes it milliseconds accuracy, then test something like "123ms" and see it is correct (instead of a truncated "120ms" etc)

Also good to test the range. For example, if theoretically it can handle 100yr then test it

Anyway this is optional, and only do it if it is convenient :)